packages/docs/src/pages/tutorial/store/basic/index.mdx

```diff
@@ -3,20 +3,21 @@ title: Storing State
 layout: tutorial
 ---
 
-Applications need to store state to be useful. (Otherwise, they are just static pages.) 
+Qwik tracks application state for two reasons:
```

Comment:
```suggestion
Applications need to store state to be useful, otherwise they're just static pages.

Qwik tracks application state for two reasons:
```